data/subscription.go

```diff
@@ -6,98 +6,48 @@ import (
 	"github.com/RocketChat/marketplace-go/events"
 )
 
-// SubscriptionStatus defines the current subscription status
+// SubscriptionStatus defines the current subscription status.
 type SubscriptionStatus string
 
-var (
-	// SubscriptionStatusTrialing is when the subscription is in the trial phase
-	SubscriptionStatusTrialing SubscriptionStatus = "trialing"
-
-	// SubscriptionStatusActive is when the subscription is active and being billed for
-	SubscriptionStatusActive SubscriptionStatus = "active"
-
-	// SubscriptionStatusCancelled is when the subscription is inactive due to being canceled
-	SubscriptionStatusCancelled SubscriptionStatus = "cancelled" //nolint:misspell //cancelled is how we spell it
-
-	// SubscriptionStatusCancelling is when the subscription is being canceled
-	SubscriptionStatusCancelling SubscriptionStatus = "cancelling" //nolint:misspell //cancelled is how we spell it
```

Comment:
we might need to suppress this rule or it might need to remain a commend like this to keep linter from being unhappy about this

Reply:
I think I kept it, but still not able to pass the lint

Reply:
https://github.com/RocketChat/marketplace-go/pull/10/files#diff-c565ff88ab8ef00dea54bf2f33c5b191cf7b1247e549cc186c508a73d5e72505R15

based on right below it seems to be gone